### DISCUSSION TITLE
Combine pseudo state

### DISCUSSION THREAD

**Discussion by ledahu05** [chakra-ui/chakra-ui#3010] [Q&A] [answered]
Hello,

I would like to style the hover state of a button when it is disabled. In the following example, the button is automatically disabled by chakra-ui when it is loading:

`<Button
              variant='primary'
              type='submit'
              w='100%'
              h='48px'
              isLoading={true}
              _disabled={{ _hover: { bgColor: 'black.100' } }}
            >
              Login
 </Button>`

I didn't find the syntax on how to do that in the doc, does anyone can help me ?

Thanks!

Christ

**Comment by ledahu05** [comment, marked as answer]
Sorry, I just realized i inversed _hover and _disabled. Everything is fine!
`<Button variant='primary' type='submit' w='100%' h='48px' isLoading={true} _hover={{ _disabled: { bgColor: 'black.100' } }} > Login </Button>`

**Reply by horatiorosa** [reply]
this is what I was looking for and what the official docs don't make obvious (unless I overlooked them). 

TYSM.
